src/com/zombiecastlerush/building/Castle.java

```diff
@@ -0,0 +1,46 @@
+package com.zombiecastlerush.building;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+class Castle {
+    //create a map of rooms in castle
+    private Map<Room, List<Room>> castleRooms = new HashMap<Room,List<Room>>();
```

Comment:
Minor quibble, fix spacing. I suggest maybe `new HashMap<Room, List<Room>>();` for consistency sake. Non-blocking.